src/hgvs/variantmapper.py

```diff
@@ -98,18 +96,14 @@ def __init__(
     # ############################################################################
     # g⟷t
 
-    def g_to_t(
-        self, var_g, tx_ac, alt_aln_method=hgvs.global_config.mapping.alt_aln_method
-    ):
+    def g_to_t(self, var_g, tx_ac, alt_aln_method=hgvs.global_config.mapping.alt_aln_method):
         if var_g.type not in "gm":
-            raise HGVSInvalidVariantError(
-                "Expected a g. or m. variant; got " + str(var_g)
-            )
+            raise HGVSInvalidVariantError("Expected a g. or m. variant; got " + str(var_g))
 
         if self._validator:
             self._validator.validate(var_g)
         var_g.fill_ref(self.hdp)
-        mapper = self._fetch_AlignmentMapper(
```

Comment:
I suspect that you made this change in order to pacify ruff's N802 warning. 

Throughout biocommons code, names are preserved without modification. I'd like to keep that convention for consistency.

Please approve and merge https://github.com/biocommons/hgvs/pull/815 into main, then merge main into this branch, and then revert the name changes.